### ISSUE TITLE
Requirements conflict

### ISSUE THREAD

**Issue by me21** [zauberzeug/nicegui#80]
I tried to upgrade my virtual environment with `pip-tools` and got a requirements conflict:

```
There are incompatible versions in the resolved dependencies:
  uvicorn==0.17.6 (from nicegui==0.8.12->-r requirements.in (line 1))
  uvicorn>=0.18.3 (from justpy==0.2.8->-r requirements.in (line 2))
```

Currently I have justpy==0.2.3 and nicegui==0.8.10 which do not conflict.
This probably means you should restrict some justpy versions in your requirements.

**Comment by falkoschindler** [comment]
@rodja I think we could upgrade to `uvicorn>=0.18.0`, don't you think?

Since uvicorn 0.18.0 now prioritizes watchfiles over watchgod, this could fix #62.

https://github.com/encode/uvicorn/blob/master/CHANGELOG.md#0180---2022-06-23

**Comment by rodja** [comment]
I updated the dependencies in the latest release [0.8.13](https://github.com/zauberzeug/nicegui/releases/tag/v0.8.13). In my tests uvicorn 0.18 did not break anything. Neither does upgrading JustPy from [0.2.3 to 0.2.8](https://github.com/justpy-org/justpy/compare/v0.2.3...v0.2.8). Later versions are not tested because JustPy undergoes a lot of refactorings which may lead to instability and unforeseeable issues.
